File: src/test/java/KebabTest.java
import org.junit.Test;

import static org.assertj.core.api.Assertions.assertThat;

public class KebabTest {

    @Test
    public void should_do_some_kebab_magic() {
        Kebab kebab = new Kebab();

        assertThat(kebab.isMagical()).isTrue();
    }
}

File: src/test/kotlin/KebabTest.kt
import org.assertj.core.api.Assertions.assertThat
import org.junit.Test

class KebabTest {

    @Test
    fun should_do_some_kebab_magic() {
        val kebab = Kebab()

        assertThat(kebab.isMagical()).isTrue();
    }
}